web/src/app/main/components/ToolbarTitle.js

```diff
@@ -94,9 +95,10 @@ NameLoader.propTypes = {
 function ToolbarTitle() {
   const width = useWidth()
   const classes = useStyles()
+  const [applicationName] = useConfigValue('General.ApplicationName')
 
   const renderTitle = (title) => {
-    document.title = `GoAlert - ${title}`
+    document.title = `${applicationName || document.title} - ${title}`
```

Comment:
This is going to be recursive if the config value is empty; we should fall back to the title on page-load rather than the current title. Maybe add `applicationName` next to `pathPrefix` in `index.html`